contrib/inventory/ec2.py

```diff
@@ -397,6 +397,14 @@ def read_settings(self):
                     continue
                 self.ec2_instance_filters[filter_key].append(filter_value)
 
+        # Instance Filters can be AND'd or OR'd based on the filters_operator
+        # parameter, default behavior is OR'd
+        self.ec2_filter_operator = 'or'
+        if config.has_option('ec2', 'filters_operator'):
+            self.ec2_filter_operator = config.get('ec2', 'filters_operator')
+            if self.ec2_filter_operator not in ('or', 'and'):
+                self.ec2_filter_operator = 'or'
```

Comment:
Would be nice to warn the user their configuration is being ignored because the specified value wasn't recognized.